README.md

```diff
@@ -20,7 +22,7 @@ Prop.write do |key, value|
 end
 ```
 
-You can choose to rely on whatever you'd like to use for transient storage. Prop does not do any sort of clean up of its key space, so you would have to implement that manually should you be using anything but an LRU cache like memcached.
+Prop does not expire it's used keys, so use memcached or similar, not redis.  
```

Comment:
let's just say it's not fine ... you can do that, but then you'd have to
know what the correct expire time is ... or just do 7 days or so ... but
I'll leave that to the imagination of the reader :)

On Tue, Oct 20, 2015 at 5:23 PM, Gary Grossman <notifications@github.com>
wrote:

> In README.md
> <https://github.com/zendesk/prop/pull/11#discussion_r42571659>:
>
> > @@ -20,7 +22,7 @@ Prop.write do |key, value|
> >  end
> >  ```
> >
> > -You can choose to rely on whatever you'd like to use for transient storage. Prop does not do any sort of clean up of its key space, so you would have to implement that manually should you be using anything but an LRU cache like memcached.
> > +Prop does not expire it's used keys, so use memcached or similar, not redis.
>
> Isn't Redis fine as long as your writer function invokes
> redis.expire(interval) as well as redis.incr(key, value)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/prop/pull/11/files#r42571659>.
>